pyviz/cmd.py

```diff
@@ -0,0 +1,102 @@
+import os
+import sys
+import argparse
+from argparse import RawTextHelpFormatter
+
+import io, zipfile
+try:    from urllib.request import urlopen
+except: from urllib import urlopen
+
+
+def download_sample_data(args):
+    import pyviz.download_sample_data
+    pyviz.download_sample_data.main(args.destination)
+
+
+
+description = """
+Command line interface for pyviz.
+
+This utility makes the pyviz examples available in the current directory
+and downloads the required sample data.
+
+The pyviz command supports the following options:
+"""
+
+epilog="""
+Example usage
+-------------
+
+$ pyviz --install-examples pyviz-examples
+
+This will place the examples in the 'pyviz-examples' directory.
+
+$ pyviz --download-sample-data data
+
+This will download the sample data to the 'data' directory.
+"""
```

Comment:
Presumably `description` and `epilog` should be the first things in the file, as they function as docstrings for the file.

Reply:
Agreed. This PR is still WIP.